jekyll/_posts/2020-10-16-version-140-released.md

```diff
@@ -0,0 +1,1076 @@
+---
+title: "Version 1.4.0 released"
+author: The Nim Team
+---
+
+We are very proud to announce Nim version 1.4 after six months of continuous development!
+Other than version 1.0, this is probably the biggest Nim release yet and we're very excited to release it!
+
+It contains exactly 900 new commits which have not already been backported to our previous versions.
+There are several new features and standard library additions compared to 1.2.
+We tried to keep breaking changes to a minimum, but some bugfixes weren't possible
+without making those necessary changes, and we feel that our users will benefit
+from them.
+
+We would recommend to all of our users to upgrade and use version 1.4.
+
+
+
+# Installing Nim 1.4
+
+## New users
+
+Check out if the package manager of your OS already ships version 1.4 or
+install it as described [here](https://nim-lang.org/install.html).
+
+
+## Existing users
+
+If you have installed a previous version of Nim using `choosenim`,
+getting Nim 1.4 is as easy as:
+
+```bash
+$ choosenim update stable
+```
+
+If you don't have `choosenim`, you can follow the same
+[install link](https://nim-lang.org/install.html) as above.
+
+
+
+# Contributors to v1.4
+
+Our contributors are amazing, and there are
+[far too many](https://github.com/nim-lang/Nim/graphs/contributors?from=2020-04-03&to=2020-10-16&type=c)
+to list here.
+Big thanks to all of you, we couldn't have pulled off this release without you!
+
+
+
+# New features
+
+## GC: ORC
+
+A new reference counting algorithm called ARC was first shipped with Nim 1.2.
+If you haven't watched it already, we recommend
+[this video from FOSDEM](https://www.youtube.com/watch?v=yA32Wxl59wo)
+where Araq explains details behind ARC and shows some benchmarks that
+illustrate the benefits.
+
+As far as we know, ARC works with the complete standard library except for
+the current implementation of `async`, because that introduces cycles that
+ARC doesn't deal with.
+There are other `async` implementations in development that don't introduce cycles.
+
+ORC is the existing ARC algorithm plus a cycle collector, and `--gc:orc` is our
+main new feature for this release!
```

Comment:
```suggestion
main new feature for this release! In the future, we plan for ORC to be Nim's
default memory management strategy.
```
Or something similar. I think this helps answer "why does this even matter?"

Reply:
This was in the original version, but we decided to remove it on purpose.

Reply:
In case it doesn't end up happening like that?